zarr/convenience.py

```diff
@@ -1179,4 +1179,7 @@ def open_consolidated(store, metadata_key='.zmetadata', mode='r+', **kwargs):
     meta_store = ConsolidatedMetadataStore(store, metadata_key=metadata_key)
 
     # pass through
-    return open(store=meta_store, chunk_store=store, mode=mode, **kwargs)
+    chunk_store = kwargs.pop('chunk_store', None)
+    if chunk_store is None:
+        chunk_store = store
```

Comment:
I think we can make this a little more concise 

```suggestion
    chunk_store = kwargs.pop('chunk_store', store)
```

Reply:
This will not work when `chunk_store=None` is  in `kwargs`.

Reply:
```suggestion
    chunk_store = kwargs.pop('chunk_store', None) or store
```

should though. :-)

Reply:
I see... didn't realize this was a competition in one-liners. :-)

Reply:
I'm normally a fan of readability over one-liners but here the "english"-ness of the statement makes it _more_ readable than otherwise. "Get `chunk_store` from kwargs, if we don't have anything then make it `store`). 

Thank you for the contribution! Agree with @jrbourbeau that a test would be nice, then this should be merged. 😀

Reply:
This is now in the new commit.